dev_guide/deployments.adoc

```diff
@@ -247,7 +247,7 @@ triggers:
 === Image Change Trigger
 
 The ImageChange trigger results in a new deployment whenever the value of an
-image stream tag changes.
+image stream tag changes either by a build or because it was imported.
```

Comment:
> end this sentence with something like "if a configchangetrigger exists."

Since we restored the old behavior, this is no longer true. You can have automatic deployments just with an image change trigger.